developer/src/kmc-keyboard/test/fixtures/basic.txt

```diff
@@ -273,13 +326,81 @@ block(keys)                         # struct COMP_KMXPLUS_KEYS {
   c0 00 00 00  00 00 00 00  index(strNull,strKey1,2)  01 00 00 00   # KMX_DWORD vkey, mod, to, flags;
   31 00 00 00  00 00 00 00  index(strNull,strKey2,2)  01 00 00 00   # KMX_DWORD vkey, mod, to, flags;
 
+# ----------------------------------------------------------------------------------------------------
+# layr
+# ----------------------------------------------------------------------------------------------------
+
+block(layr)                         # struct COMP_KMXPLUS_LAYR {
+  6c 61 79 72                       #   KMX_DWORD header.ident;   // 0000 Section name - layr
+  sizeof(layr)                      #   KMX_DWORD header.size;    // 0004 Section length
+  01 00 00 00                       #   KMX_DWORD listCount
+  01 00 00 00                       #   KMX_DWORD layerCount
+  01 00 00 00                       #   KMX_DWORD rowCount
+  02 00 00 00                       #   KMX_DWORD keyCount
+  00 00 00 00                       #   KMX_DWORD reserved0
+  00 00 00 00                       #   KMX_DWORD reserved1
+  # list 0
+  00 00 00 00                       #   KMX_DWORD flags
+  index(strNull,strUs,2)            #   KMXPLUS_STR hardware;
+  00 00 00 00                       #   KMX_DWORD layer;
+  01 00 00 00 # count
+  7B 00 00 00                       #   KMX_DWORD minDeviceWidth; // 123
+  # layers 0
+  index(strNull,strBase,2)          #   KMXPLUS_STR id;
+  00 00 00 00                       #   KMXPLUS_STR mod str
+  00 00 00 00                       #   KMX_DWORD row index
+  01 00 00 00                       #   KMX_DWORD  count
+  # rows 0
+  00 00 00 00                       #   KMX_DWORD key index
+  02 00 00 00                       #   KMX_DWORD count
+  # keys
+  index(strNull,strHmaqtua,2)       #   KMXPLUS_STR locale;       // 'hmaqtua'
+  index(strNull,strThat,2)          #   KMXPLUS_STR locale;       // 'that'
+
+# ----------------------------------------------------------------------------------------------------
+# list
+# ----------------------------------------------------------------------------------------------------
+
+# TODO-LDML: lots of comment-out ahead. Need to revisit.
```

Comment:
@mcdurdin For some reason the `index()` operators weren't quite working for me for list, so there are some hard coded values in this section. Hence the TODO.